model-providers/openai/azure-openai/runtime/src/main/java/io/quarkiverse/langchain4j/azure/openai/runtime/AzureOpenAiRecorder.java

```diff
@@ -317,6 +324,14 @@ private static ConfigValidationException.Problem createConfigProblem(String key,
                 NamedConfigUtil.isDefault(configName) ? "." : ("." + configName + "."), key));
     }
 
+    private static <T> boolean isAuthProviderAvailable(SyntheticCreationalContext<T> context, String configName) {
+        if (!NamedConfigUtil.isDefault(configName)) {
```

Comment:
Is it correct or should it be `configName == null` check ? Asking because `ModelAuthProvider.resolve` users `modelName != null` check and it is a `configName` to `ModelAuthProvider.resolve`, maybe `ModelAuthProvider.resolve` needs to be fixed ?

Reply:
@geoand It is not related to this PR (and this config name specific check here has gone) but if you can clarify it then I can follow up with a PR tuning it for `ModelAuthProvider.resolve` if necessary